portfolio/src/main/webapp/script.js

```diff
@@ -43,14 +43,59 @@ function deleteData() {
     });
 }
 
+// Create the script tag, set the appropriate attributes
+var script = document.createElement('script');
+script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyDjhk-U7NK6nQwmqSScCuZvO3l3_Q88Kps';
+script.defer = true;
+script.async = true;
+
+// Attach your callback function to the `window` object
+window.initMap = function() {
+  // JS API is loaded and available
+};
+
+// Append the 'script' element to 'head'
+document.head.appendChild(script);
+
 function createMap() {
+    document.head.appendChild(script);
   const map = new google.maps.Map(
   document.getElementById('map'),
-  {center: {lat: 37.422, lng: -122.084}, zoom: 16});
-}
+  {center: {lat: 35.2271, lng: -80.8431}, zoom: 10});
+
+const trexMarker = new google.maps.Marker({
+    position: {lat: 35.235661, lng: -80.903183},
+    map: map,
+    title: 'My High School'
+  });
+
+const hibachiSpot = new google.maps.Marker({
+    position: {lat: 35.190220, lng: -80.921910},
+    map: map,
+    title: 'This where I had lunch last week'
+  });
+
+  const bossyBeulah = new google.maps.Marker({
+    position: {lat: 35.233580, lng: -80.876500},
+    map: map,
+    title: 'Best place to have chicken sandwiches in Charlotte also conviently 5 mins near me XD'
+  });
 
+    trexMarker.addListener('click', function() {
+          map.setZoom(15);
+          map.setCenter(trexMarker.getPosition());
+        });
 
 
+    hibachiSpot.addListener('click', function() {
+          map.setZoom(15);
+          map.setCenter(hibachiSpot.getPosition());
+        });
 
 
+    bossyBeulah.addListener('click', function() {
```

Comment:
can you make sure the lines are consistently indented?

Pick a number of spaces/tabs to indent to (4 is probably good) and use it everywhere you indent.

There are also tools that can help make this eaiser (I don't know if cloud shell has autoformat, but in vim for instance, there is gg=G)

Reply:
solved